virl2_client/models/lab.py

```diff
@@ -142,6 +142,9 @@ def __init__(
         self._session = session
         self._owner = username
         self._state = None
+        self._staging_enabled = False
+        self._staging_start_remaining = True
+        self._staging_abort_on_failure = False
```

Comment:
@marpauli-cisco do we need to have all 3 as standalone properties?  I think that we have an object in UI, API and exported topologies.  Also, we should call it `node_staging` even though the name will likely change,